### DISCUSSION TITLE
AR Development

### DISCUSSION THREAD

**Discussion by charlieikosi** [pinpointltdpng/ODFD-PROJECT#9] [Ideas]
Thanks @anapitalai for suggesting [AR.js](https://ar-js-org.github.io/AR.js-Docs/location-based/) for consideration in our AR development for the project. I've gone  through the documentation and is impressed that VR /AR experiences can be created using it. Most of all, it's open source and requires no install. @anapitalai could you explore more in this pace and document what you learn?

**Comment by anapitalai** [comment]
Thanks,
I am currently doing that.

Cheers

On Wed, Apr 5, 2023 at 2:19 PM charlieikosi ***@***.***>
wrote:

> Thanks @anapitalai <https://github.com/anapitalai> for suggesting AR.js
> <https://ar-js-org.github.io/AR.js-Docs/location-based/> for
> consideration in our AR development for the project. I've gone through the
> documentation and is impressed that VR /AR experiences can be created using
> it. Most of all, it's open source and requires no install. @anapitalai
> <https://github.com/anapitalai> could you explore more in this pace and
> document what you learn?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/charlieikosi/ODFD-PROJECT/discussions/9>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABUMTLMABQHPY5GQWKA2OP3W7TXFNANCNFSM6AAAAAAWTRNWYM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Alois Ignatius Napitalai
IT Lab Manager
Surveying and Land Studies Department
PNG University of Technology
PMB 411
Lae
Phone: 473 4963

https://www.google.com/+AloisNapitalai
https://www.facebook.com/alois.napitalai
https://www.facebook.com/NAPITALAIICT
***@***.***
<http://www.unitechblogspot.wordpress.com>

**Reply by charlieikosi** [reply]
Thanks @anapitalai

**Comment by anapitalai** [comment]
Links to ARjs and hikar
https://hikar.org/
https://ar-js-org.github.io/AR.js-Docs/location-based-aframe/#connecting-to-a-web-server
